.github/workflows/release-please.yml

```diff
@@ -27,8 +27,10 @@ jobs:
           check-latest: true
           registry-url: 'https://registry.npmjs.org'
         if: ${{ steps.release.outputs.releases_created }}
+      - name: Build workspaces
+        run: npx lerna run build
       - name: Install dependencies
-        run: npm ci
+        run: npm install
```

Comment:
shouldn't we use `npm ci` here?

Reply:
npm ci was failing in the prerelease workflow so I am using npm install which was working in the pre-release workflow